awx/ui/src/screens/WorkflowApproval/WorkflowApprovalDetail/WorkflowApprovalDetail.js

```diff
@@ -169,31 +193,19 @@ function WorkflowApprovalDetail({ workflowApproval }) {
         />
       </DetailList>
       <CardActionsRow>
-        {workflowApproval.can_approve_or_deny && (
-          <>
-            <Button
-              ouiaId={`${workflowApproval.id}-approve-button`}
-              aria-label={t`Approve`}
-              variant="primary"
-              onClick={approveWorkflowApproval}
-              isDisabled={isLoading}
-            >
-              {t`Approve`}
-            </Button>
-            <Button
-              ouiaId={`${workflowApproval.id}-deny-button`}
-              aria-label={t`Deny`}
-              variant="danger"
-              onClick={denyWorkflowApproval}
-              isDisabled={isLoading}
-            >
-              {t`Deny`}
-            </Button>
-          </>
-        )}
+        {workflowApproval.status === 'pending' &&
+          workflowApproval.can_approve_or_deny && (
+            <WorkflowApprovalControls
+              selected={[workflowApproval]}
+              onApprove={approveWorkflowApproval}
```

Comment:
On the `WorkflowApprovalControls` usage we call the props with prefix `handle`.

 ```          
                    onApprove={handleApprove}
                    selected={selected}
                    onHandleDeny={handleDeny}
                    onHandleCancel={handleCancel}

```

It would be to be standardize this.